modules/Robot.h

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file   Robot.h
+ * @brief  外部リソースのインスタンスを管理するクラス
+ * @author takuchi17
+ */
+
+#ifndef ROBOT_H
+#define ROBOT_H
+
+#include "MotorController.h"
```

Comment:
公式APIのヘッダをインクルードするときには、`spikeapi.h`必要なはず
MotorController のタスクで MotorController.h 内で`spikeapi.h`をインクルードすることになると思うけど、
こっちにもあって悪いものでもないと思う。
こいつが無くてもビルドが通っているのはよくわかりません。

```suggestion

#include "spikeapi.h"
#include "MotorController.h"
```